vertx-mssql-client/src/main/java/io/vertx/mssqlclient/impl/codec/MSSQLDataTypeCodec.java

```diff
@@ -168,6 +176,13 @@ private static CharSequence decodeNVarchar(ByteBuf in) {
     return in.readCharSequence(length, StandardCharsets.UTF_16LE);
   }
 
+  private static Buffer decodeBinary(ByteBuf in) {
+    int length = in.readUnsignedShortLE();
+    byte[] bytes = new byte[length];
```

Comment:
I think we could directly here create `Unpooled.buffer(length)` and then copy to it with readBytes and then wrap it with `Buffer.buffer(ByteBuf)`